### DISCUSSION TITLE
feature request: small briefing and random IFF Plan for immersion

### DISCUSSION THREAD

**Discussion by jojojung265** [Khopa/dcs_liberation#2454] [Q&A] [answered]
Is it possible to put in a very limited briefing for the flights like it is done in briefing room? And is it possible to put in some random IFF Plan for immersion only?

**Comment by Starfire13** [comment, marked as answer]
I see no one has responded to this. Please file feature requests under Issues as a feature request, not Q&A. Thanks.
